src/higley_lab_to_nwb/benisty_2022/benisty_2022_spike2ttlinterface.py

```diff
@@ -55,6 +50,25 @@ def __init__(self, file_path: FilePathType, stream_id: str = None, stream_name:
         _test_sonpy_installation()
 
         super().__init__(file_path=file_path, stream_id=stream_id, stream_name=stream_name, all_annotations=True, verbose=verbose, es_key=es_key)
+        self.stream_ids, self.stream_names = get_stream_ids_and_names(file_path=file_path)
+    def get_metadata(self) -> dict:
+        metadata = super().get_metadata()
+
+        # Device metadata
+        # TODO store the correct metadata
+        device = dict(name="AcuisitionBoard", description="Spike2 recording signals from CED (Cambridge Electronic Design)", manufacturer="Cambridge Electronic Design")
+
+        # Add groups metadata
+        metadata["Ecephys"]["Device"] = [device]
```

Comment:
I would probably keep the default device name, and just update description and manufacturer?
```
metadata["Ecephys"]["Device"][0].update(description="...", manufacturer="...")
```

Reply:
I wanted to change the name because the default is Ecephys, and it is not actually a device that records any type of ecephys signals. I need to figure out a better name anyway

Reply:
yeah I think it is `DeviceEcephys` now, would `Spike2` be more appropriate?

Reply:
I wanted to ask them the name of the actual acquisition board because Spike2 is the acquisition software. They might be using a DAQ box from National Instruments or such. I will leave the default for now and change it after I get this answer.